assets/src/scripts/monitoring-location/components/hydrograph/parameters.test.js

```diff
@@ -78,4 +104,43 @@ describe('monitoring-location/components/hydrograph/parameters module', () => {
             loadMedian: false
         });
     });
+
+    it('Expects clicking on a row will expand and contact the correct rows', function() {
```

Comment:
nit typo: 'contact' should be 'contract'

Reply:
lol. Yes, that would make more sense.